e2e/cypress/integration/slash_commands/autocomplete_spec.js

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+describe('Integrations', () => {
+    const pluginIdDemo = 'com.mattermost.demo-plugin';
+    const pluginIdJira = 'jira';
+
+    before(() => {
+        // # Initialize setup and visit town-square
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+
+            // # If Demo plugin is already enabled, uninstall it
+            cy.apiRemovePluginById(pluginIdDemo, true);
+            cy.apiRemovePluginById(pluginIdJira, true);
+        });
+
+        const demoURL = 'https://github.com/mattermost/mattermost-plugin-demo/releases/download/v0.9.0/com.mattermost.demo-plugin-0.9.0.tar.gz';
+        const jiraURL = 'https://github.com/mattermost/mattermost-plugin-jira/releases/download/v3.0.0/jira-3.0.0.tar.gz';
+
+        cy.apiInstallPluginFromUrl(demoURL, true);
+        cy.apiInstallPluginFromUrl(jiraURL, true);
+
+        cy.apiUpdateConfig({
+            PluginSettings: {
+                PluginStates: {
+                    jira: {
+                        Enable: true,
+                    },
+                    'com.mattermost.demo-plugin': {
+                        Enable: true,
+                    },
+                },
+            },
+        });
+    });
+
+    after(() => {
+        cy.apiRemovePluginById(pluginIdDemo);
+        cy.apiRemovePluginById(pluginIdJira);
+    });
+
+    it('T2829 Test an example of plugin that uses sub commands', () => {
```

Comment:
Please include full TM4J ID. Same for other `it` blocks.
```suggestion
    it('MM-T2829 Test an example of plugin that uses sub commands', () => {
```